### PR TITLE
Merge of smaller PRs

### PR DESCRIPTION
**Pull request by EmilChri**
# Description
This is just all the very small PRs merged into one

## What is added/changed/removed
//Please describe what changes were performed in this pull request

## Issue/fixes reference
- https://github.com/AAU-P5-Moodle/moodle-1/issues/175
- https://github.com/AAU-P5-Moodle/moodle-1/issues/162
- https://github.com/AAU-P5-Moodle/moodle-1/issues/148
- https://github.com/AAU-P5-Moodle/moodle-1/issues/152
- https://github.com/AAU-P5-Moodle/moodle-1/issues/153
- https://github.com/AAU-P5-Moodle/moodle-1/issues/158
- https://github.com/AAU-P5-Moodle/moodle-1/issues/164
- https://github.com/AAU-P5-Moodle/moodle-1/issues/165
- https://github.com/AAU-P5-Moodle/moodle-1/issues/161
- https://github.com/AAU-P5-Moodle/moodle-1/issues/163
- https://github.com/AAU-P5-Moodle/moodle-1/issues/154
- https://github.com/AAU-P5-Moodle/moodle-1/issues/156
- https://github.com/AAU-P5-Moodle/moodle-1/issues/159
- https://github.com/AAU-P5-Moodle/moodle-1/issues/149
- https://github.com/AAU-P5-Moodle/moodle-1/issues/155

## Testing
There should be a checkbox at "Allow only one answer". This will make the question type to radio button.
image

Check that the editing quiz does not show when importing questions

Create a quiz
Try editing the quiz through the settings page
Click save
See that the changes are made

Create a quiz
Create a question in this quiz
Create another quiz
In this quiz import the question you created previously
This should work
Now try to import the question again
This should NOT work

# Checks
- [x] Updated / added tests for these changes
- [x] PHPunit locally passed
- [x] Codesniffer locally passed
- [x] Behat locally passed

### CODE REVIEW

**Review by AlliasGhorbani — DISMISSED** [review]
Questions cant be saved, or is it just me?

**Review by AndreasBM1 — APPROVED** [review]

**Review by DeeKahy — APPROVED** [review]
i trust you guys with my entire being. i simply dont have time to review a 700 line pull request

**Review by ChristofferGamel — APPROVED** [review]

File: server/moodle/mod/livequiz/amd/src/delete_question.js [hunk 1]
```diff
@@ -16,18 +16,23 @@ export function add_delete_question_listeners(quizid, lecturerid) {
...
+                    .then((response) => {
+                        if (response.success) {
+                            list_item.remove(); // Remove the question from the list.
+                            element.remove(); // Remove the delete button.
+                            let updated_list_length = question_list.querySelectorAll("li").length;
+                            if (updated_list_length === 0) {
+                                rerender_take_quiz_button(take_quiz_url, false);
+                            }
+                        } else {
+                            alert("Cannot delete question: " + response.message);
```
Comment (od45pv) [inline server/moodle/mod/livequiz/amd/src/delete_question.js:32]: The response for this should be more saying. Right now its "Cannot delete **answer** with participations". @Peterrahr is on it on a new PR

**Review by od45pv — APPROVED** [review]
I've tested all the issues.  There's only a minor issue on the pop-up text on #149. However this will be resolved in a later PR.

**Pull request merged**